src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String SUPPLY_STRING = "supply";
+    private static final String BUY_STRING = "buy";
+    private static final String RESULT_STRING = "result";
+    private static final String COMMA = ",";
+    private static final int ARRAY_FIRST_ELEMENT = 0;
+    private static final int ARRAY_SECOND_ELEMENT = 1;
```

Comment:
Maybe we can find more better name? 
If it is an index of the value, then `VALUE_INDEX`? Or something like this?